athenet/algorithm/derest/activation.py

```diff
@@ -0,0 +1,27 @@
+"""Functions that for any neuron and given range of inputs of layers, calculate
```

Comment:
Trolling ','
"Functions that for any neuron and given range of inputs of layers calculate neuron's range of activation." sounds better